src/core/scene/a-scene.js

```diff
@@ -693,13 +699,15 @@ class AScene extends AEntity {
    */
   tick (time, timeDelta) {
     var i;
+    var it = this.behaviorIterators;
```

Comment:
Expand variable name. iterator?

Reply:
It's so common to refer to iterators as `it` or `iter` to the point that `iterator` would harm readability IMHO. At the very least it would make the following for-loop too unwieldy, though it could as well become a while loop.

Let's first settle on the name for `behaviorIterators` as that would impact this local name as well.

Reply:
I was not familiar with it. It might be in your context / environment.
We always avoid acronyms (only exception we have is `el` for element because of how pervasive it is). a while loop should be fine. Still not convinced with the `iterator` name that seems loaded and used in other contexts but need to think about an alternative. You're probably in a better mind space than me atm to come up with a better name